src/content/en/updates/2016/07/capture-stream.md

```diff
@@ -0,0 +1,104 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The captureStream() method enables a MediaStream to be captured from a canvas, audio or video element, on Android and desktop.
+
+{# wf_updated_on: 2016-10-20 #}
+{# wf_published_on: 2016-10-20 #}
+{# wf_tags: canvas,chrome53,gif,media,security,webrtc #}
+{# wf_featured_image: /web/updates/images/2016/07/autoplay/featured.jpg #}
+
+<style>
```

Comment:
I'm **strongly** discouraging styles added unless absolutely necessary. Why do you want/need this? 

Note - there is a bug with video width, so adding `video { max-width: 100%; }` is fine.